packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/create-remote-file-node/index.js

```diff
@@ -249,71 +161,14 @@ async function processRemoteNode({
   ext,
   name,
 }) {
-  const pluginCacheDir = cache.directory
-  // See if there's response headers for this url
-  // from a previous request.
-  const cachedHeaders = await cache.get(cacheId(url))
-
-  const headers = { ...httpHeaders }
-  if (cachedHeaders && cachedHeaders.etag) {
-    headers[`If-None-Match`] = cachedHeaders.etag
-  }
-
-  // Add htaccess authentication if passed in. This isn't particularly
-  // extensible. We should define a proper API that we validate.
-  const httpOpts = {}
-  if (auth?.htaccess_pass && auth?.htaccess_user) {
-    headers[`Authorization`] = `Basic ${btoa(
-      `${auth.htaccess_user}:${auth.htaccess_pass}`
-    )}`
-  }
-
-  // Create the temp and permanent file names for the url.
-  const digest = createContentDigest(url)
-  if (!name) {
-    name = getRemoteFileName(url)
-  }
-  if (!ext) {
-    ext = getRemoteFileExtension(url)
-  }
-
-  const tmpFilename = createFilePath(pluginCacheDir, `tmp-${digest}`, ext)
-
-  // Fetch the file.
-  const response = await requestRemoteNode(url, headers, tmpFilename, httpOpts)
-
-  if (response.statusCode == 200) {
-    // Save the response headers for future requests.
-    await cache.set(cacheId(url), response.headers)
-  }
-
-  // If the user did not provide an extension and we couldn't get one from remote file, try and guess one
-  if (ext === ``) {
-    const buffer = readChunk.sync(tmpFilename, 0, fileType.minimumBytes)
-    const filetype = fileType(buffer)
-    if (filetype) {
-      ext = `.${filetype.ext}`
-    }
-  }
-
-  const filename = createFilePath(
-    path.join(pluginCacheDir, digest),
-    String(name),
-    ext
-  )
-
-  // If the status code is 200, move the piped temp file to the real name.
-  if (response.statusCode === 200) {
-    await fs.move(tmpFilename, filename, { overwrite: true })
-    // Else if 304, remove the empty response.
-  } else {
-    processingCache[url] = null
-    totalJobs -= 1
-
-    bar.total = totalJobs
-
-    await fs.remove(tmpFilename)
-  }
+  const filename = await fetchRemoteFile({
```

Comment:
This doesn't have a `cacheKey`, was this intentional?

Reply:
Talking to Tyler about how to get this inside Wordpress properly

Reply:
Removed wordpress for now